test/functional/wallet_avoidreuse.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
-# Copyright (c) 2018 The Bitcoin Core developers
+# Copyright (c) 2018-2019 The Bitcoin Core developers
```

Comment:
Generally don't touch this: maintainers run a script once a year to update this for all files.

Reply:
Done